dotCMS/src/main/java/com/dotcms/publisher/ajax/RemotePublishAjaxAction.java

```diff
@@ -351,18 +352,39 @@ public void retry ( HttpServletRequest request, HttpServletResponse response ) t
                 auditHistory.setNumTries( 0 );
                 publishAuditAPI.updatePublishAuditStatus( configStatic.getId(), status.getStatus(), auditHistory, true );
 
+                Publisher staticPublisher;
+
                 try{
-                    awss3Publisher.init(configStatic);
-                    awss3Publisher.process(null);
+
+                    List<Environment> environments = APILocator.getEnvironmentAPI().findEnvironmentsByBundleId(bundleId);
+
+                    for (Environment environment : environments) {
+
+                        List<PublishingEndPoint> endPoints = APILocator.getPublisherEndPointAPI().findSendingEndPointsByEnvironment(environment.getId());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 153 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)